### DISCUSSION TITLE
Moving container images from Docker Hub to GitHub Container Registry

### DISCUSSION THREAD

**Discussion by tomkerkhove** [tomkerkhove/promitor#1445] [Deprecations]
As announced, we are [moving container images from Docker Hub to GitHub Container Registry](https://github.com/tomkerkhove/promitor/issues/1444) and will no longer publish newer versions to Docker Hub after v2.0.0.

Learn more on:
- [Our changelog](https://changelog.promitor.io/)
- Our new Docker images on [GitHub Container Registry](https://github.com/tomkerkhove?tab=packages&repo_name=promitor&ecosystem=container)

**Comment by svrooij** [comment]
I've seem this issue in other repositories. We choose to push to both registries from Github actions. That way you don't impact the old users and you can migrate all the docs to use the new repository.
Building is the part that takes time not tagging it for both and then pushing to both.

See the actions in https://Github.com/svrooij/sonos2mqtt for a demo.

**Reply by tomkerkhove** [reply]
That certainly is true, but we chose to no longer use Docker Hub and to centralize on GitHub Container Registry.
